app/pages/lab-pages-editor/icons/DrawingToolIcon.jsx

```diff
@@ -0,0 +1,62 @@
+import PropTypes from 'prop-types';
+
+export default function DrawingToolIcon({
+  alt,
+  className = 'icon',
+  color = 'currentColor',
+  size = 24,
+  thickness = 5,
+  type
+}) {
```

Comment:
Very clever file! It'd be cool if we created a shared area for all these SVG icons defined in JS once this gets moved into FEM. I've seen a couple across the codebase and it'd be nice to know what resources are available like this.

Reply:
Thanks! I agree, it'd be good to organise these SVG icons better once Pages Editor gets moved to FEM. The Drawing Tool icons should definitely be something that sits in a shared, uh... "lib-icons" package, maybe?

Right now, in FEM, they're buried in [lib-classifier/src/plugins/tasks/drawing/components/icons](https://github.com/zooniverse/front-end-monorepo/tree/d89db9a3863af04ce11106788e528ccb10335186/packages/lib-classifier/src/plugins/tasks/drawing/components/icons) and you'd only know it's there when you need to dig for it.